vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -304,6 +305,88 @@
             		$('#file').prop("disabled",true);
             		$('#imageId').prop("disabled",false);
             	  });
+				
+            	var currentlyWaiting = false;
+            	var timeout;
+            	$('#slideSpace').sortable({
+	            	update: function(event,ui) {
+		            	var jsonObj = [];
+		            	var blockId;
+		            	$(this).children().each(function(index,element){
+			            	if(typeof($(element.innerHTML).filter('div').attr('data-position'))=="undefined") {
+			            		blockId=$(element).attr('id');
+			            	}else {
+			            		blockId = $(element.innerHTML).filter('div').attr('id');
+			            	}
+			            	if($(element.innerHTML).filter('div').attr('data-position') != index){
```

Comment:
why don't you just update all blocks and get rid of this if block?  if someone is changing the order at the same time, at least the order would be coherent and not a mix of both.